src/components/body/education/Education.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  Box, Grid, Paper, styled, Typography,
+} from '@mui/material';
+
+const Item = styled(Paper)(({ theme }) => ({
+  backgroundColor: theme.palette.mode === 'dark' ? '#1A2027' : '#fff',
+  ...theme.typography.body2,
+  padding: theme.spacing(1),
+  textAlign: 'left',
+  color: theme.palette.text.secondary,
+}));
```

Comment:
Should be placed in additional file in the same directory as component. Also, if you are using 'styled' the name of it should have a prefix -Styled. For ex.: ItemStyled